storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -4309,6 +4309,7 @@ private void addSpoutAggStats(TopologyPageInfo topoPageInfo, StormTopology topol
                 CommonAggregateStats commonStats = entry.getValue().get_common_stats();
                 setResourcesDefaultIfNotSet(spoutResources, entry.getKey(), topoConf);
                 commonStats.set_resources_map(spoutResources.get(entry.getKey()).toNormalizedMap());
+                entry.getValue().set_last_error(stormClusterState.lastError(topoPageInfo.get_id(), entry.getKey()));
```

Comment:
I believe the bug is here:

it puts "last-error" in the stat (wrongly),
https://github.com/apache/storm/blob/e82b9a707db38a814dbd2d8f85eeaac0fda0b75b/storm-server/src/main/java/org/apache/storm/stats/StatsUtil.java#L663

but retrieves it with "lastError"
https://github.com/apache/storm/blob/e82b9a707db38a814dbd2d8f85eeaac0fda0b75b/storm-server/src/main/java/org/apache/storm/stats/StatsUtil.java#L1907


This is a translation error from clojure to java since in clojure (1.x):
https://github.com/apache/storm/blob/1.x-branch/storm-core/src/clj/org/apache/storm/daemon/nimbus.clj#L2269

https://github.com/apache/storm/blob/1.x-branch/storm-core/src/clj/org/apache/storm/stats.clj#L915